running-a-nats-service/configuration/clustering/jetstream_clustering/administration.md

```diff
@@ -197,9 +197,24 @@ nats server raft step-down --user admin --password s3cr3t!
 
 Generally when shutting down NATS, including using Lame Duck Mode, the cluster will notice this and continue to function. A 5 node cluster can withstand 2 nodes being down.
```

Comment:
Would it make sense to remove the sentence: "A 5 node cluster can withstand 2 nodes being down" 

Looks like it may have been leftover at some point?

Reply:
Yeah, agree. Seems this is documented better already under "Stream replication factor" here: https://docs.nats.io/nats-concepts/jetstream#persistent-and-consistent-distributed-storage.